packages/waku/src/lib/vite-rsc/command.ts

```diff
@@ -8,6 +8,7 @@ import type { Config } from '../../config.js';
 import { resolveConfig } from '../utils/config.js';
 import { allowServerPlugin } from '../vite-plugins/allow-server.js';
 import { buildMetadataPlugin } from '../vite-plugins/build-metadata.js';
+import { cloudflarePlugin } from '../vite-plugins/cloudflare.js';
```

Comment:
Could you also add an entry in packages/waku/src/vite-plugins.ts with `unstable_` prefix?